webview/src/java/com/defold/webview/WebViewJNI.java

```diff
@@ -323,6 +269,12 @@ private void setVisibleInternal(WebViewInfo info, int visible)
             info.first = 0;
             WindowManager wm = activity.getWindowManager();
             wm.addView(info.layout, info.windowParams);
+
+            if (Build.VERSION.SDK_INT >= 30) {
+                WindowInsetsController windowInsetsController = info.layout.getWindowInsetsController();
+                windowInsetsController.setSystemBarsBehavior(WindowInsetsController.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE);
```

Comment:
I think it might be worth adding a comment why we call `setSystemBarBehavior`. The Android API docs:

_"Window would like to remain interactive when hiding navigation bars by calling [hide(int)](https://developer.android.com/reference/android/view/WindowInsetsController#hide(int)) or [WindowInsetsAnimationController#setInsetsAndAlpha(Insets, float, float)](https://developer.android.com/reference/android/view/WindowInsetsAnimationController#setInsetsAndAlpha(android.graphics.Insets,%20float,%20float))."_

Reply:
Although I don't see that we call hide() or setInsetsAndAlpha() anywhere? So why is this needed?

Reply:
What is the reasonable comment for it? These code is a replacement for the deprecated `systemUiVisibility` property on API level 30.
Basically it hide systemBars (status bar & navigation bar) at the time the window added and then having the usual swiping to show system bars behavior.
https://developer.android.com/develop/ui/views/layout/immersive